src/pages/projects/form/ProjectContributors.tsx

```diff
@@ -46,57 +54,73 @@ export const ProjectContributors = ({ currentProject, suggestedProjectManager }:
-                          key={contributor.identity.id}
-                          contributorIndex={contributorIndex}
-                          baseFieldName={`${name}[${contributorIndex}]`}
-                          contributor={contributor}
-                          removeContributor={isProjectManager(contributor) ? undefined : () => remove(contributorIndex)}
-                          isRekProject={thisIsRekProject}
-                        />
-                      );
-                    })}
-                  </TableBody>
-                </Table>
-              </TableContainer>
-            </ListPagination>
+            {contributors.length > 0 ? (
+              <ListPagination
+                count={contributors.length}
+                rowsPerPage={rowsPerPage}
+                page={currentPage}
+                onPageChange={(newPage) => setCurrentPage(newPage)}
+                onRowsPerPageChange={(newRowsPerPage) => {
+                  setRowsPerPage(newRowsPerPage);
+                  setCurrentPage(1);
+                }}
+                alternativePaginationText={t('common.number_of_rows_per_page')}>
+                <TableContainer sx={{ mb: '0.5rem' }} component={Paper}>
+                  <Table size="small" sx={alternatingTableRowColor}>
+                    <TableHead>
+                      <TableRow>
+                        <TableCell>{t('common.role')}</TableCell>
+                        <TableCell>{t('common.name')}</TableCell>
+                        <TableCell>{t('common.affiliation')}</TableCell>
+                        <TableCell>{t('common.clear')}</TableCell>
+                      </TableRow>
+                    </TableHead>
+                    <TableBody>
+                      {contributorsToShow.map((contributor) => {
+                        const contributorIndex = contributors.findIndex(
+                          (c) => c.identity.id === contributor.identity.id
+                        );
+                        return (
+                          <ContributorRow
+                            key={contributor.identity.id}
+                            contributorIndex={contributorIndex}
+                            baseFieldName={`${name}[${contributorIndex}]`}
+                            contributor={contributor}
+                            removeContributor={
+                              isProjectManager(contributor) ? undefined : () => remove(contributorIndex)
+                            }
+                            isRekProject={thisIsRekProject}
+                          />
+                        );
+                      })}
+                    </TableBody>
+                  </Table>
+                </TableContainer>
+              </ListPagination>
+            ) : contributorError && typeof contributorError === 'string' ? (
+              <Typography
+                sx={{ color: 'error.main', marginTop: '0.25rem', letterSpacing: '0.03333em', marginBottom: '1rem' }}>
```

Comment:
Noen grunn til å overstyre letter spacing her?
```suggestion
                sx={{ color: 'error.main', marginTop: '0.25rem', marginBottom: '1rem' }}>
```

Reply:
Ja det var den som ble brukt i error message under input-feltene, så ville gjøre dem like